_docs/guides/endpoints.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Install Istio for Google Cloud Endpoints Services  
+overview: Guideline to manually integrate Google Cloud Endpoints services with Istio.
+
+order: 42
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This document provides a guideline to manually integrate Istio with existing
+Google Cloud Endpoints services. 
+
+## Before you begin
+
+You need to follow the instruction to setup an Endpoints service on GKE [instructions](https://cloud.google.com/endpoints/docs/openapi/get-started-kubernetes-engine)
+After setup, you should be able to get an API key and store it in `ENDPOINTS_KEY` environment variable and the external IP address `EXTERNAL_IP`.
+You may test the service using the following command: 
+```
```

Comment:
typically line 19 would be three backticksbash:
e.g.: https://github.com/istio/istio.github.io/blame/master/_docs/setup/kubernetes/quick-start-gke-dm.md#L74-L76